### ISSUE TITLE
explicitly delete video segment

### ISSUE THREAD

**Issue by scottlamb** [scottlamb/moonfire-nvr#39]
It'd be nice to have a way to explicitly delete a particular segment of video (potentially long before it's the oldest video), for example to redact some private event.

**Comment by dolfs** [comment]
Agreed. Do we need to keep an audit log though (so we can know that segments were deleted, as opposed to never recorded?

**Comment by scottlamb** [comment]
That distinction at least is simple: if the id is < next_recording_id, it was recorded. An audit log would be nice to distinguish explicitly deleted from auto-deleted (especially once #38 is implemented and recordings are not simply auto-deleted in FIFO order) and see who did it when via what session and peer.
